docs/install.md

```diff
@@ -17,6 +17,7 @@ This command will help you to install OpenFunction and its dependencies.
 --shipwright         For installing ShipWright.
 --sync               For installing OpenFunction Sync Runtime (To be supported).
 --upgrade            Upgrade components to target version while installing.
+--force              Forced installation.
```

Comment:
`Forced installation.` is not clear enough.
Actually, this param can be used during the install and uninstall process and it's used to ignore entering y to confirm?

So it's similar to `apt-get -y install` :
Automatic yes to prompts; assume "yes" as answer to all prompts and run non-interactively.
https://askubuntu.com/questions/672892/what-does-y-mean-in-apt-get-y-install-command

Reply:
Change `--force` to `--yes` and `-y` as a short form.